src/Tribe/Customizer/Section.php

```diff
@@ -108,24 +108,6 @@ public function create_ghost_settings( $settings = array() ) {
 	private static $instances;
 
 	/**
-	 * Method to return the Private instance of the Class
-	 *
-	 * @since 4.0
-	 *
-	 * @access public
-	 * @return Tribe__Events__Pro__Customizer__Section
-	 */
-	public static function instance( $name ) {
```

Comment:
I've tried the code making no changes at all: it works as the slug is fabricated from the class name here: https://github.com/moderntribe/tribe-common/pull/249/files#diff-3eb0fdf69a7f4eade7314aa7fb9cc151L134
The `name` parameter of the instance method is really never used, the method is called here: https://github.com/moderntribe/tribe-common/pull/249/files#diff-3eb0fdf69a7f4eade7314aa7fb9cc151R139